apps/api/src/routers/user.py

```diff
@@ -25,6 +25,13 @@ class IdentityResponse(BaseModel):
     role: Union[Role, None] = None
 
 
+def getHasDeadlinePassed() -> bool:
+    now = datetime.now(timezone.utc)
+    deadline = datetime(2024, 1, 15, 7, 59, tzinfo=timezone.utc)
```

Comment:
Chore: Since this deadline is a constant, could we extract it to maybe between `router` and `IdentityResponse`?